src/coreclr/src/nativeaot/BuildIntegration/Microsoft.NETCore.Native.Unix.props

```diff
@@ -55,7 +55,6 @@ The .NET Foundation licenses this file to you under the MIT license.
 
     <ItemGroup>
       <NativeLibrary Include="$(IlcPath)/framework/libSystem.Native$(NativeLibraryExtension)" />
-      <NativeLibrary Include="$(IlcPath)/framework/libSystem.Globalization.Native$(NativeLibraryExtension)" />
```

Comment:
We need this. We need to fix the build to start building it again.

cc @VSadov

Reply:
Just wondering - Is this going to be used or just to make the build happy? 
Libraries in coreclr hardcode "QCall" at the moment. AOT uses different libraries?

Reply:
Native AOT needs a regular .a in the package. It does not need any tables with mapping, etc. The native linker that produces the final native AOT binary will take care of stitching everything together.

> Libraries in coreclr hardcode "QCall"

"QCall" is hardcoded just for the System.Globalization.Native in coreclr. Everything else uses actual name.

Reply:
Consider out of scope for this pull request and disable testing on Linux?

Or is this as easy as adding Globalization.Native to CMakeLists.txt and treating `DllImport("QCall")` same as the other `libSystem` imports (i.e. have linker deal with it)?

Reply:
> Consider out of scope for this pull request and disable testing on Linux?

I think so.